R/pkg/R/functions.R

```diff
@@ -3890,3 +3890,23 @@ setMethod("not",
             jc <- callJStatic("org.apache.spark.sql.functions", "not", x@jc)
             column(jc)
           })
+
+#' input_file_name
+#'
+#' Creates a string column for the file name of the current Spark task.
```

Comment:
I actually find this description in Scala API quite a bit confusing - what is "Spark task" and how it has "file name"?